addons/website/static/src/snippets/s_image_gallery/001.scss

```diff
@@ -0,0 +1,259 @@
+
+.o_gallery[data-vcss="001"] {
```

Comment:
Can't you follow our s_ convention now that you make a brand new shiny snippet version?